tools/wasi/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# Fizzy: A fast WebAssembly interpreter
+# Copyright 2019-2020 The Fizzy Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+include(ProjectUVWASI)
+
+set(fizzy_include_dir ${PROJECT_SOURCE_DIR}/lib/fizzy)
+
+add_executable(fizzy-wasi wasi.cpp)
+target_compile_features(fizzy-wasi PRIVATE cxx_std_17)
```

Comment:
Not needed, as `fizzy::fizzy` has it.

Reply:
This was a copy of the cmake file from spectests, all of the tools have this line too. Should be removed then.

Reply:
Our CMake scripts are not perfect, but I still vote for removing the line.